cmd/operator/operator_main.go

```diff
@@ -18,17 +18,27 @@ limitations under the License.
 package main
 
 import (
+	"flag"
 	"os"
+	"path/filepath"
 	"time"
 
-	"github.com/platform9/decco/pkg/k8sutil"
-	"github.com/platform9/decco/pkg/spacecontroller"
 	log "github.com/sirupsen/logrus"
 	"k8s.io/client-go/kubernetes"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+
+	"github.com/platform9/decco/pkg/k8sutil"
+	"github.com/platform9/decco/pkg/spacecontroller"
 )
 
 func main() {
+	var kubeconfigFlagDefault string
```

Comment:
is there some reason this looks for a HOME env var but never a KUBECONFIG env var?

Reply:
Hey @scriptory! Good question—there is no reason that I am aware of. In this PR I am just refactoring what happened in an `init` function here: https://github.com/platform9/decco/pull/31/files/ad2419519af06102c3c94c8d49b4072a80aaab9f#diff-5310b81fd6e8963e4fa499e12e9f87aeL15-L25

Moving away from the home-rolled kubeconfig setup to use the standard way that Kubernetes uses, I considered a next step (and out of scope for this PR).